affinity/pom.xml

```diff
@@ -43,7 +43,14 @@
                 <groupId>net.openhft</groupId>
                 <artifactId>third-party-bom</artifactId>
                 <type>pom</type>
-                <version>3.19.10</version>
+                <version>3.22.1</version>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>net.openhft</groupId>
+                <artifactId>chronicle-bom</artifactId>
+                <version>2.22ea-SNAPSHOT</version>
+                <type>pom</type>
```

Comment:
I'm not sure if this is OK, this project never depended on `chronicle-bom` before

Reply:
yeah good point.... I guess it should be OK - it is only a bom dependency